README.md

```diff
@@ -1,2 +1,70 @@
-# ai-card
-Working repository for common AI Card standard
+# Common AI Card and Registry Standard
+
+Co-Authors: [ToddSegal](https://github.com/ToddSegal), [David Soria Parra](https://github.com/dsp-ant)
+
+## tl;dr
+
+A2A and MCP are collaborating on a common AI card standard to benefit the AI Service Ecosystem.
+
+Contact us via GitHub Discussions, [Issues](https://github.com/Agent-Card/ai-card/issues), and [Pull Requests](https://github.com/Agent-Card/ai-card/pulls).
+
+## What problem are we solving?
+
+There are multiple evolving standards for communication protocols between AI clients and servers. While these protocols differ in capabilities and technical architecture, there are common ecosystem needs for producers and consumers of services built on top of them. This includes discovery, verifiable metadata, and trusted identity standards.
+
+Without a common discovery standard, we see duplicative and incompatible efforts in the ecosystem, such as registry, marketplace, identity providers, UI and payments extensions, and others. This increases complexity, risk of vulnerabilities, and harms interop within the developer community.
+
+## What are we doing?
+
+In this repo, we will establish a new open standard for creating, discovering, securing, and serving AI service metadata called an **AI Card**. This will be immediately adopted as the metadata solution of choice for MCP and A2A. Other protocols are encouraged to adopt this standard as well.
+
+*Important Note:* The **AI card** standard does not change the underlying protocol architecture.
+
+### Guiding Requirements
+
+#### Core AI Card Features
+
+The project will define a schema for a secure, tamper-resistant AI Card that includes
+
+* **Common server metadata**, such as publishing company, links to documentation, description, icon, and server version.
+* **Verifiable metadata**, such as accreditations, certifications, reputational scores, and badges.
+* **Identity metadata**, such as Decentralized Identifiers (DID) or Secure Production Identity Framework For Everyone (SPIFFE).
+* **Custom metadata** used by individual producers or consumers for their own needs.
+
+#### AI Protocol Services
+
+AI Card will include `services` information indicating which AI protocols the server supports, their network addresses, and protocol-specific extension metadata such as capabilities.
+
+For example, an AI Card that supports MCP, A2A, and a custom “Foo” protocol, would have entries for each of MCP, A2A, and Foo. Each entry would contain their own specific metadata (such as “MCP tools” or “A2A skills”).
+
+This protocol-specific metadata would NOT be part of the AI Card spec.
```

Comment:
Agreed and IMHO, it is useful to delegate to other protocols how manage their own metadata.
By starting w/ MCP and A2A this will be a good test to validate this design pattern.

It would be useful to allow for extensions that can be added and deprecated. I think that people might want to have the ability to use private protocols either for testing or for enterprise specific use cases.